src/pages/FlssPrototypes/SampleLoanDisplay.vue

```diff
@@ -0,0 +1,1159 @@
+<template>
+	<www-page-minimal
+		:header-theme="headerTheme"
+		:footer-theme="footerTheme"
+		:flss-logo-url="flssLogoUrl"
+	>
+		<div class="flss-landing">
+			<kv-loading-overlay
+				v-if="loadingPage"
+				class="flss-landing__loading-page"
+			/>
+
+			<!-- TODO: Alter CTA if Checkout is ready -->
+			<flss-hero
+				:hero-area-content="heroAreaContent"
+				@add-to-basket="handleAddToBasket"
+				@jump-to-loans="jumpToLoans"
+			/>
+
+			<hr>
+
+			<section class="loan-categories section" id="flssLoanSection" ref="flssLoanSection">
+				<div class="row">
+					<div class="columns">
+						<h2 class="loan-categories__header text-center">
+							Support causes you care about.
+
+							<!--PSD TODO switch to simple dropdown tailored queries-->
+
+							<!--PSD HOWTO pass queries to dev graphql endpoint-->
+							<!-- write a query via appollo here in the page?-->
+
+							<!--PSD HOWTO pass loan ids from query to federation-->
+
+							<!--PSD placeholder page is /pages/FlssPrototypes-->
+						</h2>
+
+						<div class="loan-view-controls">
+							<flss-loan-filters
+								class="loan-view-controls__filters"
+							/>
+						</div>
+
+						<flss-loan-row
```

Comment:
Let's work together and discuss the correct path for both the loan filters and the loan row.

Reply:
yes, definitely!